app/editor/src/features/content/form/hooks/useExtractTags.ts

```diff
@@ -0,0 +1,35 @@
+import { useFormikContext } from 'formik';
+import { useLookup } from 'store/hooks';
+
+import { IContentForm } from '../interfaces';
+
+export interface IExtractTagsProps {
+  setParsedTags?: React.Dispatch<React.SetStateAction<string[]>>;
+}
+
+export const useExtractTags = ({ setParsedTags }: IExtractTagsProps) => {
```

Comment:
Separate hook to reuse on both components, I'd like to extract the passed in function too.